lib/internal/http2/core.js

```diff
@@ -2969,6 +2970,14 @@ function onErrorSecureServerSession(err, socket) {
     socket.destroy(err);
 }
 
+function updateHttp2ServerSettings(settings) {
+  if (settings == null || ObjectKeys(settings).length === 0) {
+    throw new ERR_HTTP2_INVALID_SETTING_OBJECT(settings);
+  }
```

Comment:
```suggestion
  assertIsObject(settings, 'settings');
```
This is consistent with how they are handled in other places in http2.